README.rst

```diff
@@ -5,21 +5,25 @@ Flocker |coveralls|
 
 .. |coveralls| image:: https://coveralls.io/repos/ClusterHQ/flocker/badge.png
   :target: https://coveralls.io/r/ClusterHQ/flocker
-  :alt: 'Buildbot build coverage status'
 
-Flocker is a data volume manager and multi-host Docker cluster management tool.
-With it you can control your data using the same tools you use for your stateless applications by harnessing the power of ZFS on Linux.
-This means that you can run your databases, queues and key-value stores in Docker and move them around as easily as the rest of your app.
+Flocker is an open-source Container Data Volume Manager for your Dockerized applications.
```

Comment:
"open-source" or "open source"

It'd be nice to be consistent:

```
(master)(update-readmecontent-FLOC-2365 ? =)[~/.../flocker/docs]$ ack 'open-source'
introduction/index.rst
8:Flocker is an open-source Container Data Volume Manager for your Dockerized applications.
(master)(update-readmecontent-FLOC-2365 ? =)[~/.../flocker/docs]$ ack 'open source'
faq/index.rst
52:We are interested in expanding libswarm to include support for filesystems and are talking with the various open source projects about the best way to collaborate on storage and networking for volumes.
```

Though wikipedia isn't https://en.wikipedia.org/wiki/Open_source